docs/csharp/tour-of-csharp/statements.md

```diff
@@ -38,79 +38,79 @@ The `using` statement is used to obtain a resource, execute a statement, and the
 
 The following lists the kinds of statements that can be used, and provides an example for each.
 
-***Local variable declaration***
+Local variable declaration
```

Comment:
I thought about it but it will be strange to have headings for examples, and I'm still thinking that examples should be after the meaning of the statements.